public/views/InterviewForm.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
rename the file `public/views/InterviewForm.html` to `public/views/interviewForm.html`